main.go

```diff
@@ -47,6 +51,7 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	//+kubebuilder:scaffold:scheme
+	godotenv.Load()
```

Comment:
It could be in `init` func? what do you think?

Reply:
But it is in the `init` func, I'm confused hehe

Reply:
The github code review tool troll me heheh